content/docs/guidelines/best-practices/tutorial-outline/index.md

```diff
@@ -36,7 +36,7 @@ By doing so, they save countless hours of work. Writing is no different.
 
 ## Why you should always write outlines
 
-**An outline is the basic  skeleton around which you build your write-up**. Without one, it becomes much harder to shape your content and make sure it stands.
+**An outline is the basic skeleton around which you build your write-up**. Without one, it becomes much harder to shape your content and make sure it stands.
```

Comment:
Note for some cases like this, in case you're not using it yet, we can use a markdown formatter. I usually do but forgot to run it on this PR.

I also started a tool to help with syntax, to save time and avoid errors. Just gotta find the time to write it! Or delegate it to someone.